composer_local_dev/docker_files/entrypoint.sh

```diff
@@ -28,6 +29,15 @@ if [ -f /var/local/setup_python_command.sh ]; then
     /var/local/setup_python_command.sh
 fi
 
+
+pip3 install keyring keyrings.google-artifactregistry-auth
```

Comment:
Should we only install keyring if a private index is used?

Reply:
+1, please move this so it's executed only if needed.

Reply:
My comments seem to have disappeared but this is addressed within [changes to this commit](https://github.com/GoogleCloudPlatform/composer-local-dev/pull/21/commits/45a90d27ffc3b7d554c11924d0d91bb00eeb3872)